media/pages/sk/podavanie-ziadosti/page.html

```diff
@@ -0,0 +1,125 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+
+
+<h1>Sloboda informácií: Ako žiadať o informácie</h1>
+
+<h2>Úvod</h2>
+
+<p>
+Potrebujete vedieť informáciu, o ktorej predpokladáte, že ju má verejná inštitúcia k dispozícii.
+Máte právo sa jej to spýtať, a inštitúcia je povinná vám odpovedať. V zákone o slobode informácií sa
+úrady, školy, verejné inštitúcie, a právnické osoby nimi zriadené preto volajú povinná osoba. V
+ďalšom texte pre zrozumiteľnosť budeme používať pojem inštitúcia.
+</p>
+
+<p>
+Tento návod vás prevedie tým, ako o informáciu žiadať.
+</p>
+
+<p>
+Samotné podanie je relatívne jednoduché, môžu však nastať komplikácie. Ak inštitúcia informáciu
+nemá, alebo ju nesmie sprístupniť, vydá vám o tom oficiálne rozhodnutie, ktoré príde poštou. Ak si
+myslíte, že nemajú pravdu, t.j. že informáciu majú, zatajili ju, alebo ju príliš vycenzurovali,
+môžete sa odvolať.
+</p>
+
+<h2>Podanie žiadosti</h2>
+
+<h3>Čo sa môžem pýtať?</h3>
+<p>
+V princípe sa môžete spýtať na hocičo, je však potrebné uvedomiť si dve praktické veci. Je podstatne
+lepšie dokument, ktorý požadujete, presne pomenovať. Ideálne, ak viete jeho spisovú značku alebo
+presný názov. Zjednodušíte tým prácu úradníkovi. Zároveň, v prípade sporu, viete ľahko dokázať, či
+inštitúcia dokument má alebo nie. Horšie je pýtať sa na nejakú útržkovitú informáciu.
+</p>
+
+<p>
+<u>Príklad:</u> <em>“Koľko benzínu minula vaša inštitúcia minulý rok?”</em>. Aj pri najlepšej vôli
+niekedy úradník nevie, kde by informáciu hľadal, pravdepodobne mu to zaberie viac času a možno
+dostanete odpoveď, ktorá vám nič nepovie. Omnoho lepšie je tu zamyslieť sa a pýtať si konkrétny
+dokument, napr. <em>Knihu jázd za rok 20xx</em>. Tam sa napokon dozviete, že niektoré vozidlá sú
+dieselové a teda vaša pôvodná otázka ani nebola veľmi presná. Ak je inštitúcia výskumný ústav, môže
+navyše benzín používať aj na iný účel a nie je jasné, či ste náhodou nemali na mysli toto použitie.
+Mohli by ste napr. dostať odpoveď v podobe faktúr za nákup laboratórneho benzínu.
+</p>
+
+<p>
+Aby ste otázku presnejšie sformulovali, pomôže poznať, aspoň približne, úradné procesy. Niektoré
+dokumenty alebo aspoň odkazy na ne sa dajú nájsť na stránke inštitúcie. Vždy je dobre, ak si
+pozriete, ako inštitúcia interne funguje, ako pomenúva dokumenty. Ak existuje zákon, podľa ktorého
+by mal existovať dokument/informácia, ktoré chcete, pomôže ho spomenúť.
+</p>
+
+<p>
+Môžete sa spýtať aj na konkrétnu, jasnú informáciu, napr. <em>“Aká bola cena za Systém X?“</em>.
```

Comment:
@martinmacko47 V zdrojovom subore, odkial som zobral text bolo zopar preklepov, typu ze kurzivom nebol oznaceny '?' alebo ' “'. Za `<em>` textom som este pridal "." (bodku), ak to bol koniec vety.

Reply:
@viliambalaz Ano, typograficke chyby pls poopravuj.